### PR TITLE
Chore/sof 5945

### PR DESCRIPTION
**Pull request by spectre007**
This PR concerns:
- migrating static data from JS to Yaml

### CODE REVIEW

File: applications/espresso.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+name: espresso
```
Comment (timurbazhirov) [inline applications/espresso.yml:1]: Let's add a YAML linter for the PRs pls, @chu-k already added for https://github.com/Exabyte-io/python_code/blob/main/.github/workflows/cicd.yml

File: applications/espresso.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    hasAdvancedComputeOptions: true
+  # version 6.6.0
+  - version: "6.6.0"
+    isDefault: true
+    hasAdvancedComputeOptions: true
+  # version 6.7.0
+  - version: "6.7.0"
+    isDefault: true
+    hasAdvancedComputeOptions: true
+  # version 6.8.0
```
Comment (timurbazhirov) [inline applications/espresso.yml:70]: maybe `#### version 6.8.0 ####` or something like that to make it more pronounced

File: src/js/data/application_data.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
+// eslint-disable
```
Comment (timurbazhirov) [inline src/js/data/application_data.js:1]: Let's minimize the autogenerated files, pls

File: .github/workflows/cicd.yml [hunk 4]
```diff
@@ -1,12 +1,33 @@
...
+        uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - name: Checkout actions repository
+        uses: actions/checkout@v2
+        with:
+          repository: Exabyte-io/actions
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
+          ref: 4d3e0fc415f6eb28739a331e7f44cd7ba242668f
```
Comment (timurbazhirov) [inline .github/workflows/cicd.yml:24]: Can be omitted, actions PR is merged now

Comment (spectre007) [reply .github/workflows/cicd.yml]: done

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**